samplers.go

```diff
@@ -112,6 +123,36 @@ func (s *Set) Flush() []DDMetric {
 	}}
 }
 
+func (s *Set) Export() (JSONMetric, error) {
```

Comment:
So this `Export` and `Combine` method are both things that Histograms would implement in the future as they too become mergable?